### PR TITLE
Added support for selecting DB in redis

### PR DESCRIPTION
**Pull request by aleksen**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -2,7 +2,7 @@ package main
 
 import (
 	"flag"
-	"github.com/aishraj/gohort/shortener"
+	"github.com/aleksen/gohort/shortener"
```
Comment (aishraj) [inline main.go]: I don't think that the package name should be changed :)

**Comment by aishraj** [comment]
@aleksen : Could you please change the package name back to `github.com/aleksen/gohort/shortener` in `main.go`

**Comment by aleksen** [comment]
Does that do the trick?

**Comment by aishraj** [comment]
LGTM.

**Pull request merged**